astro/src/content/docs/release-notes/index.mdx

```diff
@@ -47,11 +47,118 @@ import { YouTube } from '@astro-community/astro-embed-youtube';
 
 Looking for release notes older than 1.44.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
+<ReleaseNoteHeading version='1.60.0' releaseDate='September 10, 2025' name='Prompt Param Piranha' />
+
+### Security
+More information coming soon.
+
+### New
+* <Issue issue="2208">
+    FusionAuth supports the OIDC _prompt_ parameter! This enables various use cases, such as silent authentication, as well requesting re-authentication and consent prompting.
+
+    Thanks to all of the community members that have helped us define this requirement, and waited patiently for it! 🎉
+  </Issue>
+
+### Fixed
+* <Issue issue="2691">
+    When decommissioning a license in a FusionAuth cluster, the change may not be reflected in all cluster nodes immediately. This can provide intermittent and inconsistent access to licensed features, depending on which node is handling a request.
+  </Issue>
+
+* <Issue issue="2887">
+    A user can be inadvertently shown an error when a tenant is using Advanced Threat Detection to send rate limit emails for two-factor authentication as the rate limit is exceeded by the user's action.
+
+    The user will now be shown the correct message when this condition is met.
+  </Issue>
+
+* <Issue issue="2936">
+    When a user completes a forgot password workflow, the failed login count will be reset.
+
+    This should reduce frustration for a user that changed their password after exceeding the configured failed login count. In this scenario, if the user enters the incorrect password again they will be required to wait for the configured time period before attempting login again.
+  </Issue>
+
+* <Issue issue="2966">
+    When running in dark mode, or using a dark theme, some QR code scanners are unable to read the QR code for setting up MFA. The QR code should have a high-contrast border to allow these readers to work.
+  </Issue>
+
+* <Issue issue="3105">
```

Comment:
Not sure we need to mention this one - looking at the GH issue, I think this was working as designed. 
We did ship some code to improve various timing issues with managing the license, but I don't think this is one of them. 

Even when you're "air gapped" if we can get an out bound connection - meaning you are not actually air gapped, at least not all the time, we will download a license.  If the license server is doing the correct thing - it will still be returned the correct license. 

So it seems whatever was described in that issue - perhaps from an internal support case, is not likely what what happened. I would suggest we not mention this - it is misleading I think.